apps/chat2bridge/chat2bridge.nim

```diff
@@ -248,11 +248,15 @@ when isMainModule:
   if conf.logLevel != LogLevel.NONE:
     setLogLevel(conf.logLevel)
 
+  let natRes = setupNat(conf.nat, clientId,
+                        Port(uint16(conf.libp2pTcpPort) + conf.portsShift),
+                        Port(uint16(conf.udpPort) + conf.portsShift))
+  if natRes.isErr():
+    error "Error in setupNat", error = natRes.error
+
```

Comment:
You are calling `quit()` in the `wakubridge`, but not here - should the reaction be consistent accross apps?

Reply:
I agree indeed. However, this is consistent within the chat2bridge.nim itself. I think it is better if we raise a separate PR for that.

Reply:
@vpavlin - Next week I will create a separate PR for adding consistency in the parameters checks for all the apps.